<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeacherPage</class>
 <widget class="QDialog" name="TeacherPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QTeacherPage{
background-color:rgb(247, 255, 247);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QCommandLinkButton" name="commandLinkButton_BackLink">
     <property name="font">
      <font>
       <underline>true</underline>
      </font>
     </property>
     <property name="toolTip">
      <string>Return to title window</string>
     </property>
     <property name="text">
      <string>Back to title</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="15" column="1">
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="8" column="0">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0" colspan="4">
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="13" column="3" rowspan="3">
      <widget class="QPushButton" name="pushButton_SortStudentsbyGroup">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Roboto</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Sorts your students by group name</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-radius: 8px;
background-color: rgb(255, 230, 109);
border: 1px solid gray; 
}

QPushButton:pressed{
background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1,
														stop: 0 #BEBEBE, stop:1 #D7D7D7);
}</string>
       </property>
       <property name="text">
        <string>Sort by group</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="10" column="3">
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="0">
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="3">
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0" colspan="4">
      <widget class="QLabel" name="label_Surname">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Roboto</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="8" column="1" rowspan="3">
      <widget class="QPushButton" name="pushButton_AllEnrolledStud">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Roboto</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="tabletTracking">
        <bool>true</bool>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Shows all your students</string>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-radius: 8px;
background-color: rgb(255, 230, 109);
border: 1px solid gray; 
}

QPushButton:pressed{
background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1,
														stop: 0 #BEBEBE, stop:1 #D7D7D7);
}</string>
       </property>
       <property name="text">
        <string>   Students List        </string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="13" column="0" rowspan="3">
      <widget class="QPushButton" name="pushButton_SortStudentsByName">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Roboto</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Sorts your students by surname</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-radius: 8px;
background-color: rgb(255, 230, 109);
border: 1px solid gray; 
}

QPushButton:pressed{
background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1,
														stop: 0 #BEBEBE, stop:1 #D7D7D7);
}</string>
       </property>
       <property name="text">
        <string>Sort by surname</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1" rowspan="2">
      <widget class="QPushButton" name="pushButton_ShowStudents">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Roboto</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Shows students of your chosen discipline</string>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-radius: 8px;
background-color: rgb(255, 230, 109);
border: 1px solid gray; 
}

QPushButton:pressed{
background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1,
														stop: 0 #BEBEBE, stop:1 #D7D7D7);
}</string>
       </property>
       <property name="text">
        <string>   Discipline Students</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="comboBox_Disciplines">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Roboto</family>
         <pointsize>14</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="4">
      <widget class="QLabel" name="label_CountOfStudents">
       <property name="font">
        <font>
         <family>Roboto</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <spacer name="horizontalSpacer_10">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QTableWidget" name="tableWidget_DisciplinesInfo">
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="3">
    <widget class="QTableWidget" name="tableWidget_StudentsInfo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>281</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideRight</enum>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <property name="gridStyle">
      <enum>Qt::DashLine</enum>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>40</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>120</number>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
